CHANGELOG.md

```diff
@@ -4,17 +4,10 @@
 
 ### New Features
 
-
 ### Bug Fixes
 
-- [[#1472]](https://github.com/Azure/azure-sdk-for-c/pull/1472) Fix `az_iot_message_properties_next()` when the buffer in which the properties were kept was bigger than the length of characters in the buffer.
-
 ### Other Changes and Improvements
```

Comment:
For now, remove these headings too, to match master:
https://github.com/Azure/azure-sdk-for-c/blob/master/CHANGELOG.md#110-beta4-unreleased